sympy/ntheory/__init__.py

```diff
@@ -25,6 +25,7 @@
     continued_fraction_convergents, continued_fraction
 from .digits import count_digits, digits, is_palindromic
 from .egyptian_fraction import egyptian_fraction
+from .ecm import ecm, Point
```

Comment:
For making this private, i should remove `Point`, but how will i import `Point`  for test_ecm.py.

Reply:
It can be imported directly from ecm: `from sympy.ntheory.ecm import ecm, Point`